### DISCUSSION TITLE
How can you test updated_at was updated?

### DISCUSSION THREAD

**Discussion by ckovey** [laravel/framework#36768] [Q&A] [answered]
I'm trying to test that the `updated_at` timestamp is actually updated, but it always is coming out the same.  I've not done this level of laravel database testing before so I feel like I'm missing something.

```
    public function testUserUpdatedAtUpdates()
    {
        $now = Carbon::now();
        $user = UserWithUpdated::create([
            'email' => 'test@test.com',
        ]);

        sleep(1);

        $user->email = 'test@example.com';
        $user->save();

        $this->assertLessThan($now->toDateTimeString(), $user->fresh()->updated_at->toDateTimeString());
    }
```

Which results with this failed assertion

```
Failed asserting that '2021-03-26 12:46:46' is less than '2021-03-26 12:46:46'.
```

**Comment by crynobone** [comment, marked as answer]
Use `wasChanged()`?

**Reply by ckovey** [reply]
That does show true, but there is no way to show the actual timestamp value changed?  The timestamps change when I use tinker on a real world application model

edit: actually that seems to be sufficient, thank you
